src/main/java/core/basesyntax/service/exceptions/UserNullException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.exceptions;
+
+public class UserNullException extends RuntimeException {
```

Comment:
Create tons of new exceptions for every single check - bad idea. 
Use one type of exception if it's similar check 
```suggestion
public class UserRegistrationException extends RuntimeException {
```